src/web-animations-next-player.js

```diff
@@ -30,6 +30,38 @@
 
   // TODO: add a source getter/setter
   scope.Player.prototype = {
+    _updateChildTiming: function(childPlayer, offset) {
+      if (childPlayer.startTime != this.startTime + offset) {
+        if (this.startTime === null) {
```

Comment:
> Maybe while we're moving this we could look at swapping the order of these tests - if this.startTime == null, then this.startTime + offset = offset, which could potentially equal childPlayer.startTime, even if childPlayer.currentTime != this.source.player.currentTime - offset.

Will look into it. I will also double-check to see if it's reasonable to remove all Player.proto members from animation-constructor.js.